Project/Engine/src/sysFrame/Object/Light/PointLight/PointLight.cpp

```
@@ -0,0 +1,28 @@
+#include "imgui.h"
+
+void RawPointLight::DefaultSetting() {
+    type_ = LightType::Point;
+	light_.color = {1, 1, 1, 1};
+	light_.position = {0, 2, 0};
+	light_.intensity = 0.f;
+	light_.radius = 10;
+	light_.decay = 1;
+}
+
+void RawPointLight::ImGuiSetting() {
```

Cppcheck (2.10-2)

[style] The function 'ImGuiSetting' is never used.

(unusedFunction)